osschallenge/forms.py

```diff
@@ -108,7 +106,7 @@ class ProjectForm(ModelForm):
     mentors = forms.ModelMultipleChoiceField(
         label=_('Choose  mentors'),
         required=True,
-        queryset=User.objects.filter(profile__role_id=MENTOR_ID),
+        queryset=User.objects.filter(groups__id=2),
```

Comment:
Instead of group id 2 you could do User.objects.filter(groups__name='Mentor') - more readable.